source/slang/core.meta.slang

```diff
@@ -957,12 +957,12 @@ for (int tt = 0; tt < kTypeCount; ++tt)
 
 __generic<T>
 __intrinsic_type($(kIROp_ConstantBufferType))
-__magic_type(ConstantBuffer)
+__magic_type(ConstantBufferType)
 struct ConstantBuffer {}
```

Comment:
One nice cleanup might be to allow for the `__magic_type` modifier to be used without an operand, in which case it would derive the corresponding C++ class name as `<nameOfDecl> + Type`.